apiary.apib

```diff
@@ -212,7 +212,8 @@ In the cases where "serviceId" is part of URL "SERVICE_NAME" field at body can b
                 "SERVICE_ADMIN_PASSWORD":"passwd",
                 "SERVICE_ADMIN_TOKEN":"token",
                 "SUBSERVICE_NAME":"test_subservice",
-                "NEW_SUBSERVICE_DESCRIPTION":"testsubservice new desc"
+                "NEW_SUBSERVICE_DESCRIPTION":"testsubservice new desc",
+                "NEW_SUBSERVICE_NAME":"TheNewtestsubserviceName"
```

Comment:
Apart from Apiary do we have some other documentation (markdown files) describing the API?

Reply:
Maybe a tool to translate from apiary to markdown would help us

Reply:
I was asking in order to see a good place to warn about the limitations of the renaming operations (in line with [this comment](https://github.com/telefonicaid/orchestrator/issues/330#issuecomment-1157527722)).

I understand we don't have a proper place, so wit the warning shown at Portal should suffice...

NTC